README.md

```diff
@@ -86,16 +86,16 @@ before submit the code check the Java format with the following command on the r
 ./gradlew verifyGoogleJavaFormat
 ```
 
-It any error are reported please run the following command to try to fix it
+If any error are reported please run the following command to try to fix them
```

Comment:
```suggestion
If any error is reported please run the following command to fix them
```

No try here if the syntax it is correct the linter make the job alone :-) (lukely)